packages/site/src/components/Navigation/TableOfContents.tsx

```diff
@@ -163,7 +163,7 @@ export const TableOfContents = ({
     <div
       ref={tocRef}
       className={classNames(
-        'fixed xl:article-grid article-grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px] z-20',
+        'fixed xl:article-grid article-grid-gap xl:w-screen xl:pointer-events-none overflow-auto max-xl:min-w-[300px]',
```

Comment:
@rowanc1 reverting this causes two issues: the menu items can be covered by the toolbar div and the inset black background no longer blacks out the compute toolbar. The toolbar is set to `z-10` to lift above content in the main body.

![image](https://github.com/executablebooks/myst-theme/assets/1473646/909922d9-e998-4a0e-a5c2-aa85ab575cf5)

did the `z-20` here interfere with something else?

Reply:
or is there a better way to deal with this overall?

Reply:
It breaks the cross-references which appear under the toc.

The pointer-events-none should allow the controls to be mostly fixed though, only the right hand side is active?

Reply:
Left some other notes on this here:
https://github.com/executablebooks/myst-theme/pull/136#discussion_r1251169214